jobfunnel/config/proxy.py

```diff
@@ -0,0 +1,30 @@
+"""Proxy configuration for Session()
+"""
+from jobfunnel.config import BaseConfig
+
+
+class ProxyConfig(BaseConfig):
+    """Simple config object to contain proxy configuration
+    """
+    def __init__(self, protocol: str, ip_address: str, port: int) -> None:
```

Comment:
Not an urgent matter, but it'd be nice in the future to have some error checking here for ip address, port, etc. And I realize you do have a `validate` method. Silly me :sweat_smile:

Reply:
I added a schema validation for this, in the cerberus schema, but I gotta find a way to do that validation here as well. 

Realistically i need to use the cerberus schema validation on the constructed config dict in cli parsing to accomplish this, but i need to ensure the errors are going to misdirect the user too badly. 

But tricky but on my mind and agreed we should add this for the cli case.